pravega/pravega
# Fixed io.pravega.test.integration.ReadTest.testReceivingReadCall is failing very frequently

**Change log description**  
Fixed io.pravega.test.integration.ReadTest.testReceivingReadCall is failing very frequently.

**Purpose of the change**  
Fixed io.pravega.test.integration.ReadTest.testReceivingReadCall by using separate thread pool for this test class, ealier it was using threadpool of AppendTest class and somerimes this threadpool was getting terminated before submitting tasks of testReceivingReadCall test case, and it was failing with IllegalStateException. 

**What the code does**  
Uses its own threadpool in test case testReceivingReadCall to create channel.

**How to verify it**  
run gradle integration test(i.e. command `**./gradlew test:integration:test --parallel**
`) and see this test gets passes or not.
